docs/docs-content/clusters/edge/local-ui/cluster-management/export-cluster-definition.md

```diff
@@ -24,7 +46,115 @@ host. A cluster definition contains one or more cluster profiles, including the
+   development environment.
+
+   ```bash
+   mv palette-edge /usr/local/bin
+   ```
+
+3. Verify the installation of the Palette Edge CLI by issuing the following command. The output will display information
+   about the currently supported OS and Kubernetes distributions.
+
+   ```bash
+   palette-edge show
+   ```
+
+   ```hideClipboard bash
+   # Sample output
+   ┌────────────────────────────────────────────────────────────────────────┐
+   | OS Flavor     | Description        | Base Image URI                    |
+   | opensuse-leap | Opensuse Leap 15.4 | quay.io/kairos/core-opensuse-leap |
+   | ubuntu-20     | Ubuntu 20.4 LTS    | quay.io/kairos/core-ubuntu-20-lts |
+   | ubuntu-22     | Ubuntu 22.4 LTS    | quay.io/kairos/core-ubuntu-22-lts |
+   └────────────────────────────────────────────────────────────────────────┘
+   ┌─────────────────────────────────────────────────────────────────────────────────────────────┐
+   | K8S Flavor | Description        | Supported Versions                                        |
+   | k3s        | Rancher K3s        | 1.25.2-k3s1,1.24.6-k3s1,1.23.12-k3s1,1.22.15-k3s1         |
+   | kubeadm    | Kubernetes kubeadm | 1.25.2,1.24.6,1.23.12,1.22.15                             |
+   | rke2       | Rancher RK2        | 1.25.2-rke2r1,1.24.6-rke2r1,1.23.12-rke2r1,1.22.15-rke2r1 |
+   └─────────────────────────────────────────────────────────────────────────────────────────────┘
+   ┌─────────────────────────────────┐
+   | Component             | Version |
+   | Spectro Agent Version | v3.4.3  |
+   | Kairos Version        | v2.0.3  |
+   └─────────────────────────────────┘
+   ```
+
+4. Log in to [Palette](https://console.spectrocloud.com).
+
+5. Select the project you want to deploy the Edge host to and copy down the **Project ID**. You can find the project id
+   at the top right side corner of the landing page below the **User drop-down Menu**.
+
+6. Navigate to the left **Main Menu** and select **Profiles**.
+
+7. Use the **Cloud Types drop-down Menu** and select **Edge Native**.
+
+8. Click on the cluster profile you want to include in the cluster definition.
+
+9. You can find the cluster profile ID by reviewing the URL of the current page. The cluster profile ID is the last
+   value in the URL. Repeat this step for all the cluster profiles you want to include in the cluster definition.
+
+   ```text
+   https://console.spectrocloud.com/projects/yourProjectId/profiles/cluster/<YourClusterProfileHere>
+   ```
+
+10. Issue the following command to export the cluster definition.
+
+    ```shell
+    palette-edge-cli build --api-key <apiKey> \
+    --project-id <projectId> \
+    --palette-endpoint <paletteEndpoint> \
+    --cluster-definition-name <clusterDefinitionName> \
+    --cluster-definition-profile-ids <clusterDefinitionProfileId1,clusterDefinitionProfileId2,...> \
+    --cloud-type edge-native
```

Comment:
We are missing a definition for the `cloud-type` flag.

Reply:
Ah, this one is no longer a required parameter because it defaults (and is almost always) edge native. I cannot think of anything at this time where you would do this for non-Edge. I will remove this one.